### ISSUE TITLE
Spells and features do not work when clicked. (5e)

### ISSUE THREAD

**Issue by gaffdninja** [Drental/fvtt-tokenactionhud#253]
Hi, guys every time I try and use a spell or a feature from the hud nothing happens, I can still roll manually from the character sheet. Skills saving throws still work. It is just Spells and features. Funnily enough, I can recharge features from the HUD but I cant use them.

**Comment by Drental** [comment]
please share this issue with developers, that understand the 5e system. the V10 migration of the module is something that I can not handle alone, For now all I got running is the structure as a concept, but the system implementations cannot be maintained by me.

**Comment by gaffdninja** [comment]
https://www.loom.com/share/df33b3b434c5426f9ef6fb9b78ab809e

I don't think it's a big issue. because everything else seems to be working fine. It's just features and spells and now attacking as well.

**Comment by gaffdninja** [comment]
https://www.loom.com/share/2e06a3b285bd4370879f883571c53503

**Comment by Oshcakes** [comment]
Hey Drental! This only happens with midi-qol enabled. Here is the console error:

```
commons.js:1940 Uncaught (in promise) TypeError: Cannot set property type of #<Event> which has only a getter
[Detected 4 packages: midi-qol, lib-wrapper, system:dnd5e, token-action-hud]
    at _mergeInsert (commons.js:1940)
    at mergeObject (commons.js:1914)
    at _mergeUpdate (commons.js:1956)
    at mergeObject (commons.js:1913)
    at Item5e.doItemUse (itemhandling.js:479)
    at Wrapper.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:620)
    at Item5e.🎁CONFIG.Item.documentClass.prototype.use#0 (libWrapper-wrapper.js:189)
    at Item5e.roll (item.mjs:681)
    at RollHandlerBase5e.rollItemMacro (bundle.min.js:1)
    at RollHandlerBase5e._handleMacros (bundle.min.js:1)
    at RollHandlerBase5e.doHandleActionEvent (bundle.min.js:1)
    at RollHandlerBase5e.handleActionEvent (bundle.min.js:1)
    at handleClick (bundle.min.js:1)
    at HTMLDivElement.<anonymous> (bundle.min.js:1)
    at HTMLDivElement.dispatch (jquery.min.js:2)
    at HTMLDivElement.y.handle (jquery.min.js:2)
```

**Comment by Larkinabout** [comment]
This should be fixed in 3.0.1 as item#roll is replaced with item#use. I've tested as working with and without midi-qol enabled.

One thing I noticed is that if you have targetting required in midi-qol, item uses don't seem to trigger a warning to target a token, so you'll just need to make sure you're targetting when using items from the HUD's inventory. Spells do trigger a targetting warning.

**Comment by Oshcakes** [comment]
Yes this is fixed in 3.0.1

**Comment by gaffdninja** [comment]
Yey ty guys. Thanks for doing the work it is very much appreciated.
